Answers.md

```diff
@@ -1,3 +1,16 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
 
 For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+1. A Docker container creates a copy of the development enviorment so the server can run the app in the same enviorment. A Virtual machine is clone of an OS ontop of the current OS sharing resourses.
+
+2. 49160:8080 are the port numbers for Localhost:HTTP 
+
+3. Kubermetes is and automated system used to deploying and scaling the containers.
```

Comment:
To elaborate a bit more, container orchestration platforms allow developers to more easily provision a whole slew of Docker containers and specify how each should coordinate with each other and how resources should be shared between all the containers.